lib/promoted/ruby/client/util.rb

```diff
@@ -0,0 +1,13 @@
+module Promoted
+    module Ruby
+      module Client
+        module Util
+            def self.translate_args(args)
+                args.transform_keys(&:to_s).transform_keys(&:to_underscore).transform_keys(&:to_sym)
+              rescue => e
+                raise 'Unable to parse args. Please pass correct arguments. Must be JSON'
```

Comment:
side-comment - This might also be mis-communication with the contractor.  I don't know Ruby norms.  He probably took whatever chats with me and assumed we need a lot of flexibility.

If Symbol Hash are the norm, we can have the client just do that and then ask other clients to convert (even provide a utility for it).

Reply:
Inclined to go with it.